etc/component-config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "interception": {
+    "turenDidWakeUp": ["custodian"]
```

Comment:
I think interception key/name requires namespace to guarantee uniqueness, look like `'turen.didWakeUp'`, that will make the event system more robust.

Reply:
And if every single component can define its event configuration, it will make component development more decoupled.

Reply:
The names of the intercept-able event are now used as method name in components like https://github.com/yodaos-project/yodart/pull/184/files#diff-68613b8e01793b850b0eade0561edcc2R181. 

Names like `turen.didWakeUp` could not be used as method name *directly*, though it's possible through computed properties.

Reply:
It can be defined like this:
``` {
''interception": {
  "turen": {
    "didWakeUp": ["custodian.turenDidWakeUp", "anotherComponent"]
  }
}
} ```
the event will be dispatched to `anotherComponent.didWakeUp` by default, and  "custodian.turenDidWakeUp" for custom.

Reply:
Or if we defined this in component, not global, it can be:
```
{
  "interception": {
    "turen": {
      "didWakeUp": "turenDidWakeUp"
    }
  }
} 
```

Reply:
Prefixes in name e.g. `turenDidWakeUp` already stands for namespaces. 

Doesn't get your point on
> ```
> {
>   "interception": {
>     "turen": {
>       "didWakeUp": "turenDidWakeUp"
>     }
>   }
> } 
> ```

Reply:
`turenDidWakeUp ` can't be unique in global, this is just an **appointment**, not a **rule**
. The component developer can define any name for it. But if we introduce the namespace, we can circumvent this situation from **rule**.

Reply:
> can't be unique in global, this is just an appointment, not a rule.

Yea, it just is an appointment, the former might grant more rights to developer, but simple enough. See https://github.com/yodaos-project/yodart/issues/183#issuecomment-450576544 for details.

Reply:
If our goal is to **make component development more flexible and free**, then the necessary rules are worth discussing. The simple approach may actually make developers feel confused.

Reply:
`
the former might grant more rights to developer, but simple enough.
`
namespace doesn't add much complexity, but it makes the event definition more explicit. We just need to guarantee the uniqueness of the component name, then everything will be fine.
In fact, I prefer this format:
`
{
  "interception": {
    "turen": {
      "didWakeUp": "turenDidWakeUp"
    }
  }
} 
`
:)

Reply:
Config by component in global are weird a bit.